docs/resources/llama-stack-spec.yaml

```diff
@@ -5130,195 +4972,3 @@ x-tagGroups:
       - ToolRuntime
       - VectorDBs
       - VectorIO
-  - name: Types
-    tags:
-      - AgentCandidate
-      - AgentConfig
```

Comment:
Is these x-tagGroups deleted on purpose?

Reply:
@yanxi0830 yes, on purpose. It was just noise. There is no point to generating separate Tags for component schemas.